src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,89 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.LinkedHashMap;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String csvSplitBy = ",";
+    private static final int operationTypeIndex = 0;
+    private static final int operationAmountIndex = 1;
+    private static final String resultingRowName = "result";
+    private static final String buyRowName = "buy";
+    private static final String supplyRowName = "supply";
+    private LinkedHashMap<String, Integer> hashtable = new LinkedHashMap<>();
+    private List<String> linesFromCsvFile;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        fillHashTable(fromFileName);
+        calcResultRow();
+        writeStatToFile(toFileName);
+    }
```

Comment:
Use this structure it will looks more simple 
![image](https://github.com/mate-academy/jv-work-with-file/assets/63095184/0f78632a-f6ce-4e06-9e79-3812b47280ac)